packages/app/src/components/catalog/shared/RunBooksCard.tsx

```diff
@@ -28,10 +32,19 @@ export const RunBooksCard = () => {
   };
   const getAlertData = (arg: any) => {
     setAlertData(arg);
+  };
+
+  useEffect(() => {
     if (alertData?.annotations.runbook_url) {
-      getRunbook(alertData.annotations.runbook_url);
+      const runbookUrl = new URL(alertData.annotations.runbook_url);
+      let url = alertData.annotations.runbook_url;
+      if (runbookUrl.hostname === 'github.com') {
+        url = convertGithubUrlToRaw(alertData.annotations.runbook_url);
+      }
+      getRunbook(url);
```

Comment:
I think you can make this more streamlined.

```suggestion
      const url = alertData.annotations.runbook_url;
      getRunbook(new URL(url).hostname === 'github.com' ? convertGithubUrlToRaw(url) : url);
```

If you don't like the above, at least do something like:


```suggestion
      let url = alertData.annotations.runbook_url;
      const runbookUrl = new URL(alertData.annotations.runbook_url);
      if (runbookUrl.hostname === 'github.com') {
        url = convertGithubUrlToRaw(alertData.annotations.runbook_url);
      }
      getRunbook(url);
```

or:


```suggestion
      const runbookUrl = new URL(alertData.annotations.runbook_url);
      if (runbookUrl.hostname === 'github.com') {
        getRunbook(convertGithubUrlToRaw(alertData.annotations.runbook_url));
      } else {
        getRunbook(alertData.annotations.runbook_url);
      }
```

Reply:
Great idea :+1: like it